app/assets/stylesheets/active_admin_sidebar.scss

```diff
@@ -0,0 +1,87 @@
+@import "active_admin_sidebar_pure_icons";
+
+body.active_admin {
+  #active_admin_content.left_sidebar, #active_admin_content.collapsed_sidebar {
+    #sidebar {
+      display: block;
+      margin-left: 0;
+    }
+
+    .paginated_collection_contents {
+      clear: none;
+      float: left;
+      width: 100%;
+    }
+
+    .blank_slate_container {
+      clear: none;
+    }
+
+    .columns {
+      clear: none;
+    }
+
+    #main_content_wrapper {
+      float: inherit;
+      margin-left: 298px;
+      width: auto;
+      #main_content {
```

Comment:
Rule declaration should be preceded by an empty line<br>Avoid using id selectors<br>Selector should have depth of applicability no greater than 2, but was 4<br>Selector `main_content` should be written in lowercase with hyphens